webhooks/ratio.go

```diff
@@ -0,0 +1,90 @@
+package webhooks
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	inf "gopkg.in/inf.v0"
```

Comment:
Do we really need this? By my understanding, built in types should do for what we do there.

Reply:
This is bit of a clusterfuck.

`resource.Quantity` uses this type. Initially i tried turning them into floats, but turned into a lot of casting. From `float64` to `int64`, back to `resource.Quantity`. IMO it turned out more readable by just using the same number type `resource` is using.

Reply:
Ah, makes sense.